packages/ui/src/views/repo/repo-summary/repo-summary.tsx

```diff
@@ -99,12 +99,18 @@ export function RepoSummaryView({
   const { t } = useTranslationStore()
   const { repoId, spaceId, selectedBranchTag } = useRepoBranchesStore()
 
-  if (loading) return <SkeletonList />
+  if (loading)
+    return (
+      <SandboxLayout.Main fullWidth>
+        <SandboxLayout.Content>
+          <SkeletonList />
+        </SandboxLayout.Content>
+      </SandboxLayout.Main>
+    )
```

Comment:
As a general rule, always add parens when the body of a condition can't fit on the same line as the `if`
```suggestion
  if (loading) {
    return (
      <SandboxLayout.Main fullWidth>
        <SandboxLayout.Content>
          <SkeletonList />
        </SandboxLayout.Content>
      </SandboxLayout.Main>
    )
  }
```

Makes it easier to visually scan and see the scope.